pulser-core/pulser/backend/remote.py

```diff
@@ -102,7 +133,9 @@ def submit(
         pass
 
     @abstractmethod
-    def _fetch_result(self, submission_id: str) -> typing.Sequence[Result]:
+    def _fetch_result(
+        self, submission_id: str, job_ids: list[str] | None
+    ) -> typing.Sequence[Result]:
```

Comment:
Not a related change but, how have you defined which types are imported via `typing.Type` or imported at the top? It's just the abstract classes?

Reply:
It's basically everything except `typing.Sequence` so as to not confuse it with `pulser.Sequence` (I know, it's unfortunate naming but there's no going back now)